tests/MongoDB.Driver.Core.Tests/Core/Clusters/SingleServerClusterTests.cs

```diff
@@ -148,18 +149,44 @@ public void Dispose_should_dispose_of_the_server()
             _capturedEvents.Any().Should().BeFalse();
         }
 
+        [Theory]
+        [ParameterAttributeData]
+        public void ServerDescription_type_should_be_replaced_on_Unknown_when_isMaster_setName_wrong(
```

Comment:
Rename method to:

    ServerDescription_type_should_be_replaced_with_Unknown_when_isMaster_setName_is_different